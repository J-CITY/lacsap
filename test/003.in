program HelloWorld;
var mas : array [1..10] of char;
    int : integer;
    r : real
begin
    int := 222222222;
    mas[2] := 'a';
    r := 3.1415;
    writeln('111111', mas[2]);
    writeln('');
end.
