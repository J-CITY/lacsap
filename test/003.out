ERROR: -1003 5 11
